packages/cli/src/constructs/check-group.ts

```diff
@@ -189,14 +191,21 @@ export class CheckGroup extends Construct {
     this.browserChecks = props.browserChecks
     const fileAbsolutePath = Session.checkFileAbsolutePath!
     if (props.browserChecks?.testMatch) {
-      this.__addChecks(fileAbsolutePath, props.browserChecks.testMatch)
+      this.__addChecks(fileAbsolutePath, props.browserChecks.testMatch, CheckTypes.BROWSER)
+    }
+    if (props.multiStepChecks?.testMatch) {
```

Comment:
Just checking - this property was already added in this previous PR, but it didn't do anything?

https://github.com/checkly/checkly-cli/pull/868/files#diff-5679022c8ca8164d11865b9bd6417b09260e3c4bbbf9386c468c3f53af16ccd8R36-R41

Reply:
Good catch, looks like was not in use